### ISSUE TITLE
RF Issues

### ISSUE THREAD

**Issue by jkurina86** [NOAA-PMEL/EDD-LCP_Control#7]
- We are routing the Iridium directly from the module to the antenna, so it doesn't need any connectors on the board.
- The ublox EVA-M8 series hardware integration manual (pgs 20-21) states the following regarding the antenna routing:
> - Add a ground plane underneath the GNSS module to reduce interference. This is especially important for the RF input line.
> - For improved shielding, add as many vias as possible around the micro strip/coplanar waveguide, around the serial communication lines, underneath the GNSS module, etc.
> - **The micro strip / coplanar waveguide must be 50 Ω** and be routed in a section of the PCB where minimal interference from noise sources can be expected. **Make sure that there is only GND around and under the RF line**.

These instructions were not followed and have resulted in degraded GPS performance. To resolve this we need to do the following:
- Use a copper-pour on the top layer and ensure that the ground copper layer is present beneath the RF trace to create a coplanar RF waveguide. Use via stitching on both sides of the waveguide.
- https://www.microwaves101.com/calculators/864-coplanar-waveguide-calculator use this to calculate the RF trace width and length to impedance match ZO=50 Ohms. Use ER=4.7 for FR-4 board. Substrate Thickness (H) is in mils and our boards are set to approximately 18.9 mils thick.

**Comment by noahlawrence-slavas** [comment]
Is the ground plane still needed, I was confused vs the last hallway conversation we had about this?

**Comment by jkurina86** [comment]
Yes. The Ublox datasheet mentions the ground plane beneath the trace being used in the RF calculations. Below is a cross-section illustration of a standard coplanar waveguide:

![Cross_Section_of_Coplanar_Waveguide_Transmission_Line](https://github.com/user-attachments/assets/f8d436ca-1cc0-4fac-9f3a-3116c7806ee7)
